packages/docs/src/routes/docs/(qwik)/components/tasks/index.mdx

```diff
@@ -232,6 +233,7 @@ const delay = (time: number) => new Promise((res) => setTimeout(res, time));
 
 > Sometimes it is required to only run code either in the server or in the client. This can be achieved by using the `isServer` and `isBrowser` booleans exported from `@builder.io/qwik/build` as shown above.
 
+> If you want to track `useStore()` objects and execute something when any of their internal values change, use `track(myExampleStore)` directly without the anonymous function. This prevents `track()` from losing the reference to the original reactive object.
```

Comment:
This is inaccurate. What this non-function form does is tell `track` to treat it as a signal/store and subscribe to it. The function form would have to be `track(() => Object.keys(store))`.

In any case, it still doesn't subscribe to deep changes.